kernel/aster-nix/src/fs/file_table.rs

```diff
@@ -274,6 +330,7 @@ impl Clone for FileTableEntry {
             file: self.file.clone(),
             flags: AtomicU8::new(self.flags.load(Ordering::Relaxed)),
             subject: Subject::new(),
+            owner: RwLock::new(None),
```

Comment:
I have a question, when cloning fd, will the owner be duplicated?

Reply:
Yes, you're right. I'll fix it.